server/server.go

```diff
@@ -765,3 +808,75 @@ func (s *Server) ID() string {
 	defer s.mu.Unlock()
 	return s.info.ID
 }
+
+// Event subjects
+const (
+	clientConnectEventSubj    = "_SYS.CLIENT.CONNECT"
+	clientDisconnectEventSubj = "_SYS.CLIENT.DISCONNECT"
+)
+
+func (s *Server) sendEventNotification(subject string, v interface{}) {
```

Comment:
See below, maybe it would be simply to receive a byte array. Let createXEvent() do the marshaling?